### ISSUE TITLE
http://cineblog01.fm: is a parked domain

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe#2302]
The `http://cineblog01.fm` website is now a parked domain, as it's obvious when visiting it:

<img width="689" alt="Screenshot 2022-09-13 at 12 35 24" src="https://user-images.githubusercontent.com/337298/189879992-9d4ccf83-2bfd-4135-8465-d538cba091bd.png">

However, [using the MAT for this domain](https://explorer.ooni.org/chart/mat?test_name=web_connectivity&domain=cineblog01.fm&since=2022-08-13&until=2022-09-13&axis_x=measurement_start_day), we see anomalies:

<img width="1254" alt="Screenshot 2022-09-13 at 12 36 32" src="https://user-images.githubusercontent.com/337298/189880201-d24fb5d0-4bf1-424f-87ca-6b6ec2f677a9.png">

Are those anomalies relevant or noise?

Based on [what I see from an Explorer search](https://explorer.ooni.org/search?since=2022-09-10&until=2022-09-11&test_name=web_connectivity&domain=cineblog01.fm&failure=false), the domain seems to be in the Italian test list.

I will sample three anomalies and comment on each of them.

[The first sampled anomaly](https://explorer.ooni.org/measurement/20220910T080738Z_webconnectivity_IT_12874_n1_BK90UBoC1gTSLPZ4?input=http%3A%2F%2Fcineblog01.fm%2F) is a case of censorship where the system resolver returns `127.0.0.1` in AS12874 (Fastweb SpA).

[The second sampled anomaly](https://explorer.ooni.org/measurement/20220910T104728Z_webconnectivity_IT_1267_n1_a78nZJMVmgxGaZ27?input=http%3A%2F%2Fcineblog01.fm%2F) is a case where the system resolver returns `NXDOMAIN` in AS1267 (WIND TRE S.P.A.)

[The third sampled anomaly](https://explorer.ooni.org/measurement/20220910T090723Z_webconnectivity_IT_12874_n1_WvSWdAMgbgZGAQgP?input=http%3A%2F%2Fcineblog01.fm%2F) is just like the first one.

So, there is _a little bit of residual DNS censorship_ in _some Italian ISPs_. However, the domain is now parked, so...

**Comment by agrabeli** [comment]
Thanks @bassosimone . Since this is a parked domain, I'd recommend removing it from the relevant test list. The fact that it *may* be blocked on some networks is perhaps of limited relevance, given that the site doesn't exist anymore.
